packages/terra-content-container/package.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "terra-content-container",
+  "main": "lib/ContentContainer.js",
+  "version": "0.0.0",
+  "description": "The Terra ContentContainer is a structural component for the purpose of arranging content with a header.",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/cerner/terra-core.git"
+  },
+  "keywords": [
+    "Cerner",
+    "Terra",
+    "terra-content-container",
+    "ContentContainer",
+    "UI"
+  ],
+  "author": "Cerner Corporation",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/cerner/terra-core/issues"
+  },
+  "homepage": "https://github.com/cerner/terra-core#readme",
+  "devDependencies": {
+    "terra-toolkit": "^0.x"
+  },
+  "peerDependencies": {
+    "react": "15.4.2",
```

Comment:
We probably want to relax the version constraint here to match our other packages: https://github.com/cerner/terra-core/blob/master/packages/terra-badge/package.json#L37-L38